src/test/scala/com/mozilla/telemetry/streaming/EventsToAmplitudeTest.scala

```diff
@@ -96,7 +96,7 @@ class EventsToAmplitudeTest extends FlatSpec with Matchers with BeforeAndAfterAl
        |  "event_type": "Meta - session split",
        |  "event_properties": {
        |    "subsession_length": "3600",
-       |    "active_ticks": "2000",
+       |    "active_ticks": "271",
```

Comment:
The change the expected value here to match what we were injecting as the scalar value for active_ticks.